src/components/confirm/confirm.ts

```diff
@@ -12,15 +27,15 @@ function useNativeConfirm() {
         && window.confirm;
 }
 
-async function nativeConfirm(options) {
+async function nativeConfirm(options: string | ConfirmOptions) {
     if (typeof options === 'string') {
         options = {
             title: '',
             text: options
-        };
+        } as ConfirmOptions;
     }
 
-    const text = (options.text || '').replaceAll('<br/>', '\n');
+    const text = (options.text ?? '').replace(/<br\/>/g, '\n');
```

Comment:
Replace by default will only replace one instance, but you can use a regex pattern to do a global replace.